src/core/BUILD

```diff
@@ -40,6 +40,11 @@ exports_files(
     visibility = ["//:__subpackages__"],
 )
 
+config_setting(
+    name = "maximize_threadyness",
+    values = {"define": "GRPC_MAXIMIZE_THREADYNESS=1"},
```

Comment:
To prevent some confusion between the C macro (`#define`) and the bazel define here, why don't we make this:

```
config_setting(
    name = "maximize_threadyness",
    values = {"define": "maximize_threadyness=true"},`.
}
```

The invocation would then be 
```
bazel build --config=tsan --define=maximize_threadyness=true ...
```

That follows the same pattern for how you can enable all warnings in the build, `--define=use_strict_warning=true`.

Reply:
But different then latent_see, grpc_allow_exceptions, and remote_execution